contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -165,7 +167,11 @@ contract ValidatorShare is ValidatorShareStorage {
         return _amountToSlash;
     }
 
-    function drain(address token, address payable destination, uint256 amount) external onlyOwner {
+    function drain(
```

Comment:
ValidatorShare requires drain? Don't all the funds stay in stakeManager?

Reply:
Mostly this validator share contract will be public on the validator's page (also on our wallet). It might be possible that someone from the community might send Matic tokens to a validator share contract directly (just added for safety)